exposed-core/build.gradle.kts

```diff
@@ -11,6 +11,7 @@ repositories {
 
 dependencies {
     api(kotlin("stdlib"))
+    api(kotlin("stdlib-jdk8"))
```

Comment:
Please remove that dependency as Exposed expected to be used with Java 6+.

Reply:
√